src/views/Orders.vue

```diff
@@ -395,12 +407,43 @@ export default defineComponent({
           },{
             text: header,
             handler: () => {
-              this.store.dispatch('order/packShipGroupItems', {order, part, facilityId: this.currentFacility.facilityId})
+              if(!pickup) {
+                this.packShippingOrders(order, part);
+              } else {
+                this.store.dispatch('order/packShipGroupItems', {order, part, facilityId: this.currentFacility.facilityId})
+              }
             }
           }]
         });
       return alert.present();
     },
+    async packShippingOrders(currenOrder: any, part: any) {
+      try {
+        const resp = await OrderService.packOrder({
+          'picklistBinId': currenOrder.picklistBinId,
+          'orderId': currenOrder.orderId
+        })
+
+        if(!hasError(resp)) {
+          showToast(translate("Order packed and ready for delivery"));
+          const orders = JSON.parse(JSON.stringify(this.orders));
+          const orderIndex = orders.findIndex((order: any) => {
```

Comment:
We have the similar code to remove updated order in two places. Can we have single action for this logic?